### PR TITLE
Yaml issue form

### PR DESCRIPTION
**Pull request by hober**
This is my attempt at converting the new issue template to use GitHub issue forms, as @gsnedders suggested. You can see how it renders here: https://github.com/hober/standards-positions/issues/new?assignees=&labels=&template=request-for-position.yml

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/request-for-position.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+name: Request for position
+description: Solicit WebKit's position on an emerging web specification
+body:
+  - type: markdown
+    attributes:
+      value: "## WebKittens who can provide input"
+  - type: input
+    id: webkittens
+    attributes:
```
Comment (nschonni) [inline .github/ISSUE_TEMPLATE/request-for-position.yml:9]: Forms inputs can have labels. Might look better than the H2s
```suggestion
    attributes:
      label: WebKittens who can provide input
```

File: .github/ISSUE_TEMPLATE/request-for-position.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      label: "URL to the spec"
+      placeholder: "https://www.w3.org/TR/SHORTNAME"
+    validations:
+      required: true
+  - type: input
+    id: spec-repo
+    attributes:
+      label: "URL to the spec's repository"
+      description: "On GitHub or whatever repository hosting provider the spec uses."
+      placeholder: "https://github.com/ORG/REPO"
```
Comment (marcoscaceres) [inline .github/ISSUE_TEMPLATE/request-for-position.yml:35]: As we used W3C above... 

```suggestion
      placeholder: "https://github.com/w3c/REPO"
```

File: .github/ISSUE_TEMPLATE/request-for-position.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      value: "## Bugs tracking this feature"
+  - type: input
+    id: webkit-bugzilla
+    attributes:
+      label: "WebKit Bugzilla URL"
+      placeholder: "https://webkit.org/b/NNNNNN"
+  - type: input
+    id: radar
+    attributes:
+      label: "Radar URL"
```
Comment (marcoscaceres) [inline .github/ISSUE_TEMPLATE/request-for-position.yml:70]: a lot of people won't know what this is and I wonder if we should include it at all. If there is a bugzilla bug already, there will likely be a radar linked from it.... maybe we should drop this?

**Review by marcoscaceres — APPROVED** [review]
Just a comment about the radar field... it's totally fine if it stays, but maybe worth considering dropping it.

File: .github/ISSUE_TEMPLATE/request-for-position.yml (hunk 3 above)
Comment (annevk) [reply .github/ISSUE_TEMPLATE/request-for-position.yml:70]: It hasn't been an issue with the current template I think and it's kinda useful to have it. I suggest we keep it for now.

Comment (gsnedders) [reply .github/ISSUE_TEMPLATE/request-for-position.yml:70]: imo, I think it's not a bad thing to discuss (again), but I don't think we should block changing the format of the form on this

**Comment by dbaron** [comment]
FYI, the README.md (i.e., the second paragraph of https://github.com/WebKit/standards-positions) has a "file an issue" link that links directly to `issues/new` and thus avoids the template.  I'd suggest linking either to `issues/new/choose` or (probably preferably) directly to the template version.

**Pull request merged**